### PR TITLE
feat: Use the Wearable Preview for Wearable screenshots

### LINKED ISSUE #2212: Use WearablePreview to take wearables screenshots as well
Deprecate the current way of getting wearables screenshots and use the WearablePreview to get both emotes/wearable thumbnails.

### PR DESCRIPTION
**Pull request by juanmahidalgo**
Closes #2212

image

### CODE REVIEW

File: src/components/Modals/EditPriceAndBeneficiaryModal/EditPriceAndBeneficiaryModal.css [hunk 1]
```diff
@@ -63,3 +63,12 @@
   display: flex;
   align-items: center;
 }
+
+.EditPriceAndBeneficiaryModal .checkbox.make-it-free {
```
Comment (juanmahidalgo) [inline src/components/Modals/EditPriceAndBeneficiaryModal/EditPriceAndBeneficiaryModal.css:67]: Not related to the PR, fixing some styles as a drive-by.

File: src/components/Modals/CreateSingleItemModal/utils.ts [hunk 2]
```diff
@@ -60,6 +60,6 @@ export function toWearableWithBlobs(file: File, isEmote = false): WearableWithBl
         }
       ]
     },
-    emoteDataV0: isEmote ? { loop: false } : undefined
+    ...(isEmote ? { emoteDataV0: isEmote ? { loop: false } : undefined } : {})
```
Comment (cazala) [inline src/components/Modals/CreateSingleItemModal/utils.ts:63]: is the second isEmote necessary here?

Comment (juanmahidalgo) [reply src/components/Modals/CreateSingleItemModal/utils.ts]: not at all! removed :)

**Review by cazala — APPROVED** [review]
